styles/components/alert.module.scss

```diff
@@ -0,0 +1,123 @@
+.popUpWrapper {
+    position: fixed;
+    top: 0;
+    left: 0;
+    background-color: rgba(68, 68, 68, 0.5);
+    margin: 0;
+    padding: 0;
+    opacity: 0;
+    z-index: 50;
+    width: 100%;
+    height: 100%;
+    visibility: hidden;
+    transform: scale(1.1);
+    transition: visibility 0s linear 0.25s, opacity 0.25s 0s, transform 0.25s;
```

Comment:
It seems to me that `visibility 0s linear 0.25s` is not a valid transition because the delay is mentioned twice. 
Or is it?